var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -285,6 +285,7 @@ class Mapl(CMakePackage):
     conflicts("+extdata2g", when="@:2.40.3 %intel@2021.7:")
 
     depends_on("c", type="build")
+    depends_on("cxx", type="build")
```

Comment:
I don't understand this either

Reply:
Ahh. Per MAPL CMakeLists:
```cmake
project (
  MAPL
  VERSION 2.55.0
  LANGUAGES Fortran CXX C)  # Note - CXX is required for ESMF
```

I guess when we moved to CMake we needed it? I (or someone--this is before GitHub) added a comment about needing `CXX`.

That was years ago. I wonder if MAPL needs that anymore? Still, adding it to MAPL's package is correct per our CMake, which might be wrong.